### PR TITLE
first attempt at responsive workflow design

### PR DESCRIPTION
**Pull request by aarongundel**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/templates/views/components/plugins/workflow.htm [hunk 1]
```diff
@@ -222,7 +233,7 @@
                 </div>
     
                 <!--ko if: ko.unwrap(activeStep().informationBoxData) -->
-                <div
+                <div class="step-info"
```
Comment (chrabyrd) [inline arches/app/templates/views/components/plugins/workflow.htm:236]: I'm don't see this declared anywhere?

File: arches/app/media/css/arches.scss [hunk 2]
```diff
@@ -6242,6 +6242,199 @@ th.notif-type {
...
+    .step-metadata {
+        display: flex;
+    }
+
+    .workflow-step-title,
+    .workflow-plugin .btn .succinct,
+    .ep-tools .fa-sign-out {
+        display: none
+    }
+}
```
Comment (chrabyrd) [inline arches/app/media/css/arches.scss:6268]: Can this work outside of a screen break?

**Review by chrabyrd — COMMENTED** [review]
overall looks good! Just the two questions :-)

File: arches/app/templates/views/components/plugins/workflow.htm (hunk 1 above)
Comment (aarongundel) [reply arches/app/templates/views/components/plugins/workflow.htm:236]: probably leftover cruft.

File: arches/app/media/css/arches.scss (hunk 2 above)
Comment (aarongundel) [reply arches/app/media/css/arches.scss]: yeah; I removed the break.  In this case, it was actually yanking these styles below the screen break.  This was helpful, but not really necessary.

**Review by chrabyrd — APPROVED** [review]
lgtm!

**Pull request merged**